src/components/MultipleLogo/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Currency } from '@trisolaris/sdk'
+import React from 'react'
+import styled from 'styled-components'
+import CurrencyLogo from '../CurrencyLogo'
+
+const Wrapper = styled.div<{ margin: boolean; sizeraw: number }>`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  margin-right: ${({ sizeraw, margin }) => margin && (sizeraw / 3 + 8).toString() + 'px'};
+`
+
+interface MultipleLogoProps {
+  margin?: boolean
+  size?: number
+  currencies: Currency[]
+  separation?: number
+}
+
+const HigherLogo = styled(CurrencyLogo)<{ zIndex: number }>`
+  z-index: ${({ zIndex }) => zIndex};
+  margin-left: ${({ zIndex }) => (zIndex > 3 ? '0px' : '0px')};
+  margin-right: ${({ zIndex }) => (zIndex > 3 ? '35px' : '20px')};
+`
+const CoveredLogo = styled(CurrencyLogo)<{ sizeraw: number; zIndex: number; position: number }>`
+  position: absolute;
+  z-index: ${({ zIndex }) => zIndex};
+  left: ${({ position }) => `${position}px`};
+`
+
+export default function MultipleLogo({ currencies, size = 16, margin = false, separation }: MultipleLogoProps) {
```

Comment:
Can this be refactored to also handle the logic for 2 currencies?  It'll be confusing to use one component for 2 currencies and a different component for `n` currencies